app/assets/javascripts/student_profile/LightBehaviorDetails.js

```diff
@@ -122,14 +123,19 @@ export default class LightBehaviorDetails extends React.Component {
   // color), just to mark this as different.
   renderIncidents(filteredDisciplineIncidents) {
     return (
-      <div style={{width: '50%'}}>
-        {filteredDisciplineIncidents.map(incident => (
-          <IncidentCard
-            key={incident.id}
-            style={{background: '#eee', marginBottom: 20}}
-            incidentCard={incident}
-          />
-        ))}
+      <div className="IncidentHistory" style={{display: 'flex'}}>
```

Comment:
This class name doesn't look like it's used for anything, can we remove it?  In general, let's use names to label the root element inside a component (with the same name as the component), or for things that need proper CSS, for testing or other JS interop.